addon/components/horizontal-bar-chart.js

```diff
@@ -36,10 +38,71 @@ export default ChartComponent.extend(FloatingTooltipMixin,
 
   finishedData: Ember.computed.alias('sortedData'),
 
+  // Checks xDomain to see if there is negative data
+  hasNegativeData: Ember.computed('xDomain', function(){
+    return (this.get('xDomain')[0] < 0);
+  }),
+
+  // Checks xDomain to see if there is positive data
+  hasPositiveData: Ember.computed('xDomain', function(){
+    return (this.get('xDomain')[1] > 0);
+  }),
+
   // ----------------------------------------------------------------------------
   // Layout
   // ----------------------------------------------------------------------------
 
+  /**
+  * Overrides values in addon/mixins/axis-titles.js
```

Comment:
This asterisk should move to the right by one space to align with the one above it. Do the same thing for others below.